.travis.yml

```diff
@@ -8,6 +8,16 @@ python:
 env:
     - TEST_SERVER_MODE=false
     - TEST_SERVER_MODE=true
+matrix:
```

Comment:
Can you not add 3.7 into the `python:` list above?

Reply:
It doesn't work - https://travis-ci.org/spulec/moto/builds/407918370.

Maybe I should add a comment to remind future visitors about the relevant Travis CI issue?

Reply:
Wow that's odd. Yeah add a comment, and if you dont mind, poke travis to see if they're aware of that as I'd think that should "just work"

Reply:
Updated!

Yes they aware - the "sudo + xenial" solution comes from a Travis staff - https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-401756442. Sigh.